modules/apps/frontend-js/frontend-js-state-web/src/main/resources/META-INF/resources/State.ts

```diff
@@ -16,6 +16,7 @@ import SubscriberMap from './SubscriberMap';
 import deepFreeze from './deepFreeze';
 
 import type {Immutable} from './types';
+import * as Types from 'frontend-js-web/src/main/resources/META-INF/resources/liferay/types';
```

Comment:
Not loving this 😂

Reply:
:heart_eyes:

Reply:
It would be great to add global types in a custom folder to be added via `typeRoots` to avoid importing types explicitly.

Reply:
Hmm to some extent I agree with that but it's a little weird because you wouldn't be using `Liferay` directly because they are declared globally, that's why I find it weirder because normally this behavior is so that it's visible to all files because we are adding to `window`, opposed to defining the specific types of a module. For example, it might make sense if it had a syntax like:

```typescript
import type '/path/types';
```

Maybe if the types for Liferay weren't `declare global` but that wouldn't work. Either way it would be the same thing as the global types for Window.